src/lib/cache/subCache.cpp

```diff
@@ -71,8 +71,8 @@ volatile SubCacheState subCacheState = ScsIdle;
 */
 EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityType, const std::string& _isPattern)
 {
-  entityId     = _entityId;
-  entityType   = _entityType;
+  const std::string entityId     = _entityId;
+  const std::string entityType   = _entityType;
```

Comment:
This looks a little dangerous ...

    entityId     = _entityId;

sets the member 'entityId' to have the value of '_entityId'.

On the other hand:

    const std::string entityId     = _entityId;

just created a local variable and then dies ...